pdfarranger/exporter.py

```diff
@@ -147,13 +147,12 @@ def export(input_files, pages, file_out, mode, mdata):
         angle = row.angle
         angle0 = current_page.Rotate if '/Rotate' in current_page else 0
         new_page = pdf_output.copy_foreign(current_page)
-        #new_page.Rotate = 0
+        new_page.Rotate = 0
```

Comment:
Setting new_page.Rotate = 0 and adding an optional angle argument to  _mediabox() in exporter.py is changing quite a lot and a bit of a hack.